### ISSUE TITLE
Retina support

### ISSUE THREAD

**Issue by JanTvrdik** [JanTvrdik/PhpVersionSwitcher#16]
Currently it unfortunately does not support retina.

![switcher-retina](https://cloud.githubusercontent.com/assets/175109/10880254/0b888a74-815a-11e5-9b7f-70a19f1b9efb.png)

**Comment by hrach** [comment]
IMO: icons vs app. If app is started in retina env, the UI is drown correctly, only the icons are wrongly scaled. So I would fix the icons first :)

**Comment by JanTvrdik** [comment]
Nope, the app was started in retina env and the menu items are not scaled properly.
